src/components/ha-slider.js

```diff
@@ -1,8 +1,28 @@
 import "@polymer/paper-slider";
 
 const PaperSliderClass = customElements.get("paper-slider");
+let subTemplate;
 
 class HaSlider extends PaperSliderClass {
+  static get template() {
+    if (!subTemplate) {
+      subTemplate = PaperSliderClass.template.cloneNode(true);
+
+      const superStyle = subTemplate.content.querySelector("style");
+
+      // append style to add mirroring of pin in RTL
+      superStyle.appendChild(
+        document.createTextNode(
+          "#sliderContainer.pin.expand > .slider-knob > .slider-knob-inner::after { " +
```

Comment:
Use backticks to write the CSS

```ts
document.createTextNode(`
.yourCSS {

}
`)
```

Reply:
Didn't know that one. Thanks! Fixed. Looks much better.